client/modules/driver/components/driver-assignment/RouteSummary.js

```diff
@@ -12,10 +14,16 @@ const RouteSummary = ({route}) => {
   if (hours) hourString = hours === 1 ? `${hours} hour` : `${hours} hours`
   if (minutes) minuteString = `${minutes} minutes`
 
+  ///////RONALD
+  var fixDistance = ` ${Math.round(distance / 100) / 10} km, `
```

Comment:
Use `const` for constants and `let` for variables instead of `var`
`fixDistance` isn't a great name, maybe `distanceString` since there's already `hourString` etc